docs/api-markdown-documenter/index.js

```diff
@@ -6,14 +6,19 @@
 /*
  * This index script runs `render-api-documentation.js` using the version configurations described
  * in data/versions.json.
+ * 
+ * This script accepts a boolean parameter which is saved to the isLocal constant. Passing in true
+ * will configure the script to render docs for local api content instead of the docs versions
+ * specified in versions.json.
  */
 
 const chalk = require("chalk");
 const path = require("path");
 const versions = require("../data/versions.json");
 const { renderApiDocumentation } = require("./render-api-documentation");
 
-const docVersions = versions.params.previousVersions.concat(versions.params.currentVersion);
+const isLocal = process.argv[2];
+const docVersions = isLocal ? ["local"] : versions.params.previousVersions.concat(versions.params.currentVersion);
```

Comment:
Our Hugo infra reads `versions.json` directly, if I recall correctly. So don't we need to update website logic in order to display the local API docs?